Formula/igv.rb

```diff
@@ -1,29 +1,30 @@
 class Igv < Formula
   desc "Interactive Genomics Viewer"
   homepage "https://www.broadinstitute.org/software/igv"
-  url "https://data.broadinstitute.org/igv/projects/downloads/2.8/IGV_2.8.13.zip"
-  sha256 "95c90ac4327c5bc135cd0c94f66123930bd05050f3a64271b00cca5b52ae9fc6"
+  url "https://data.broadinstitute.org/igv/projects/downloads/2.9/IGV_2.9.4.zip"
+  sha256 "05717ec6d0386a5eafe93c8b1af371c0013355eae74312b78eae5fdd99fc5fc1"
+  license "MIT"
 
   livecheck do
     url "https://software.broadinstitute.org/software/igv/download"
     regex(/href=.*?IGV[._-]v?(\d+(?:\.\d+)+)\.zip/i)
   end
 
-  bottle :unneeded
-
   depends_on "openjdk"
 
   def install
     inreplace ["igv.sh", "igvtools"], /^prefix=.*/, "prefix=#{libexec}"
     bin.install "igv.sh" => "igv"
     bin.install "igvtools"
     libexec.install "igv.args", "lib"
-    bin.env_script_all_files libexec, JAVA_HOME: Formula["openjdk"].opt_prefix
+    bin.env_script_all_files libexec, Language::Java.overridable_java_home_env
   end
 
   test do
     assert_match "Usage:", shell_output("#{bin}/igvtools")
     assert_match "org/broad/igv/ui/IGV.class", shell_output("#{Formula["openjdk"].bin}/jar tf #{libexec}/lib/igv.jar")
+
+    ENV.append "_JAVA_OPTIONS", "-Duser.home=#{testpath}"
```

Comment:
Finally found reason for previous test failure.

`igv` needs to create files/directories inside `user.home`.
The default value set by Homebrew via `_JAVA_OPTIONS` was causing problems. 
Overriding with test dir avoids issue where `igv` tries to request user input via GUI `yes/no` box.